.github/workflows/container-ci.yaml

```diff
@@ -179,7 +190,8 @@ jobs:
   test-containers:
     needs: [setup-build, setup-test]
     if: ${{ needs.setup-test.outputs.matrix != '[]' }}
-    runs-on: [self-hosted, Linux, "${{ inputs.runner_label || fromJson(needs.setup-build.outputs.matrix).runner_label }}"]
+    # runs-on: [ubuntu-latest, "${{ inputs.runner_label || fromJson(needs.setup-build.outputs.matrix).runner_label }}"]
```

Comment:
Don't forget to change this back after you are done

Reply:
Will make the changes after I have done revalidating.